frontend/www/js/omegaup/components/UpdatedDatePicker.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <b-input-group size="16">
```

Comment:
It looks like that size is not being applied because the styles from `b-input-group` are being used, which cause the content to expand across the entire container div. So it can be removed:
```suggestion
  <b-input-group >
```